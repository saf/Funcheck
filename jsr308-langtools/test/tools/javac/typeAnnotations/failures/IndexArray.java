/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary test indexing of an array
 * @author Mahmood Ali
 * @compile/fail/ref=IndexArray.out -XDrawDiagnostics -source 1.7 IndexArray.java
 */
class IndexArray {
  int[] var;
  int a = var @A [1];
}

@interface A { }
