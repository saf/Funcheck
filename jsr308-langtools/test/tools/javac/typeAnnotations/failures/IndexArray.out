IndexArray.java:10:15: compiler.err.illegal.start.of.expr
1 error
